reference/tools/gnu/pkgconfig.rst

```diff
@@ -35,6 +38,50 @@ Use the ``pc`` file information to fill a ``cpp_info`` object:
         pkg_config.fill_cpp_info(self.cpp_info, is_system=False, system_libs=["m", "rt"])
 
 
+However, ``PkgConfig`` will invoke the ``pkg-config`` executable to extract the information from the ``.pc`` file.
+The ``pkg-config`` executable must be available in the system path for this case, otherwise, it will fail when installing the consumed package.
+
+
+Using pkg-config from Conan package instead of system
```

Comment:
I'd say using ``pkg-config`` from a Conan ``tool_requires``.

Reply:
Done, changes addressed to the commit https://github.com/conan-io/docs/pull/3811/commits/c3139187527d33139748bc9dda08c6ba74a2acb1